<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_status">
       <attribute name="title">
        <string>工作状态监测</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="7" colspan="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_card1_acoustic_speed"/>
        </item>
        <item row="7" column="3">
         <widget class="QPushButton" name="pushButton_card1_acoustic_speed_test">
          <property name="text">
           <string>测试</string>
          </property>
         </widget>
        </item>
        <item row="7" column="6">
         <widget class="QLineEdit" name="lineEdit_card2_acoustic_speed"/>
        </item>
        <item row="7" column="7">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>m/s</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_region1">
          <property name="text">
           <string>MAPUTO侧</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_card1_online">
             <property name="text">
              <string>在线</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_card1_offline">
             <property name="text">
              <string>离线</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="11" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>采集卡2参数</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>传感器1位置</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_stream1">
          <property name="text">
           <string>上游</string>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QLineEdit" name="lineEdit_reqion2"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>上下游</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>监测区域</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>监测区域</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>上下游</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_card1_sensor1_location">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="6" rowspan="2" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_card2_online">
             <property name="text">
              <string>在线</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_card2_offline">
             <property name="text">
              <string>离线</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QLineEdit" name="lineEdit_stream2"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>传感器2位置</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineEdit_card1_sensor2_location">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="5">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>传感器1位置</string>
          </property>
         </widget>
        </item>
        <item row="9" column="5">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>传感器2位置</string>
          </property>
         </widget>
        </item>
        <item row="8" column="6">
         <widget class="QLineEdit" name="lineEdit_card2_sensor1_location"/>
        </item>
        <item row="9" column="6">
         <widget class="QLineEdit" name="lineEdit_card2_sensor2_location"/>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="8" column="7">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="9" column="7">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>采集卡1参数</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_card1_cable_length"/>
        </item>
        <item row="6" column="6">
         <widget class="QLineEdit" name="lineEdit_card2_cable_length"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>缆索长度</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>缆索长度</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="6" column="7">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>缆索声速</string>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>缆索声速</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>m/s</string>
          </property>
         </widget>
        </item>
        <item row="7" column="8">
         <widget class="QPushButton" name="pushButton_card2_acoustic_speed_test">
          <property name="text">
           <string>测试</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="4">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="9">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="pushButton_card1_config_ok">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item row="10" column="5">
         <widget class="QPushButton" name="pushButton_card2_config_ok">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_card1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>采集卡1监测</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_figure">
          <item row="1" column="0">
           <widget class="PlotWidget" name="GLW_spectrum"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>记录</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="PlotWidget" name="GLW_localization"/>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QTextEdit" name="textEdit_record"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="PlotWidget" name="GLW_waveform"/>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_card2">
       <attribute name="title">
        <string>采集卡2监测</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_figure_2">
          <item row="1" column="0">
           <widget class="PlotWidget" name="GLW_spectrum_2"/>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>记录</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="PlotWidget" name="GLW_localization_2"/>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QTextEdit" name="textEdit_record_2"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="PlotWidget" name="GLW_waveform_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_capture">
    <property name="title">
     <string>采集设置</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_startcapture"/>
    <addaction name="action_stopcapture"/>
    <addaction name="action_paramconfig"/>
   </widget>
   <widget class="QMenu" name="menu_analysis">
    <property name="title">
     <string>分析设置</string>
    </property>
    <addaction name="action_showhis"/>
   </widget>
   <addaction name="menu_capture"/>
   <addaction name="menu_analysis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_startcapture">
   <property name="text">
    <string>开始采集</string>
   </property>
  </action>
  <action name="action_paramconfig">
   <property name="text">
    <string>参数配置</string>
   </property>
  </action>
  <action name="action_showhis">
   <property name="text">
    <string>查看历史</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>参数配置</string>
   </property>
  </action>
  <action name="action_cableparam">
   <property name="text">
    <string>缆索参数</string>
   </property>
  </action>
  <action name="action_stopcapture">
   <property name="text">
    <string>停止采集</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
